hash/cdtv.xml

```diff
@@ -2630,7 +2630,7 @@ cd32: hangs at ReadySoft logo
 	<!-- Xenon2Megablast -->
 	<software name="xenon2" supported="no">
 		<!-- Also has "wnazennjn" language option, which is a pseudo-Polish joke -->
-		<description>Xenon 2 - Megablast (Eng, Fra, Ger, Ita)</description>
```

Comment:
Now the wnazennjn comment above lost the context. Either remove it or add an extra line stating something like: 
`<!-- English, French, German, Italian supported language options -->`

Reply:
Shouldn't it be "Europe" rather than "Euro", in any case?

Reply:
> Now the wnazennjn comment above lost the context. Either remove it or add an extra line stating something like: `<!-- English, French, German, Italian supported language options -->`

Added the language info
`<info name="language" value="English/French/German/Italian" />`

Reply:
> Shouldn't it be "Europe" rather than "Euro", in any case?

It is mentioned in lots of software lists "Euro", so I didn't change it.
Is it worth the changing?

Reply:
Most of the softlists use "Euro" instead of "Europe". I've tried to be consistent per file. Not sure if it's worth the trouble to change them, but sometimes people get bored 😄

Reply:
The non-standard abbreviations should be phased out.  Euro is a [marsupial](https://en.wikipedia.org/wiki/Common_wallaroo) and a [currency](https://en.wikipedia.org/wiki/Euro).